keys/src/address.rs

```diff
@@ -6,7 +6,7 @@ use nimiq_hash::{hash_typed_array, Blake2bHash, Blake2bHasher, Hasher};
 use nimiq_macros::create_typed_array;
 use thiserror::Error;
 
-use crate::{key_pair::KeyPair, PublicKey};
+use crate::{key_pair::KeyPair, EdDSAPublicKey, PublicKey, WebauthnPublicKey};
```

Comment:
I agree; we should rename `WebauthnPublicKey` to something specific about the curve/scheme.

Reply:
Would you prefer `ECDSAPublicKey` for the new name, or `ES256PublicKey`? I tend to the later one, as it describes the whole thing including the key length, not just the scheme. 

In a similar vain, `EdDSAPublicKey` should probably be named `Ed25519PublicKey` instead. Or `SchnoorPublicKey`, as it's already renamed in some imports?

Reply:
Renamed them now to `Ed25519PublicKey` and `ES256PublicKey` respectively. I find that represents their properties best.